packages/pwa/app/utils/responsive-image.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import theme from '@chakra-ui/theme'
```

Comment:
This theme value is Chakra's default theme, it's not necessarily the theme that the project is using. You can see that we are extending the theme in `app/theme/index.js`. Although we aren't extending the `breakpoints` in our theme, and using the defaults, there is a pretty decent change that someone will want to change those values through theme extension. 

This means that our component won't react properly to those changes. An idea you can look into is to use the breakpoints from the actual theme as an argument. You'd probably have to get the theme via the `useTheme` hook and use it in your component and pass the breakpoint through `getResponsiveImageAttributes`.

Reply:
Good catch! I've now used `useTheme` to get the app theme's breakpoints.